graylog2-server/src/main/java/org/graylog/security/authzroles/AuthzRolesResource.java

```diff
@@ -126,7 +180,53 @@ public AuthzRoleDTO get(@ApiParam(name = "roleId") @PathParam("roleId") @NotBlan
             throw new BadRequestException("Invalid argument in search query: " + e.getMessage());
         }
 
-        final PaginatedList<AuthzRoleDTO> result = authzRolesService.findPaginatedForUser(searchQuery, page, perPage,sort, order, username);
+        final PaginatedList<AuthzRoleDTO> result = authzRolesService.findPaginatedForUser(
+                searchQuery, page, perPage,sort, order, username);
         return PaginatedResponse.create("roles", result, query);
     }
+
+    @PUT
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation("Add a user to a role")
+    @AuditEvent(type = AuditEventTypes.USER_UPDATE)
+    @Path("{roleId}/assignee/add/{username}")
```

Comment:
I think we can use `PUT <role-id>/assignee/<username>` here:
```suggestion
    @Path("{roleId}/assignee/{username}")
```